src/Forms/GridField/GridFieldViewButton.php

```diff
@@ -65,7 +65,7 @@ public function getColumnContent($field, $record, $col)
             return null;
         }
         $data = new ArrayData([
-            'Link' => Controller::join_links($field->Link('item'), $record->ID, 'view')
+            'Link' => $this->getURL($field, $record, $col);
```

Comment:
Please fix the syntax error
```suggestion
            'Link' => $this->getURL($field, $record, $col),
```

Reply:
Oops! Sorry about that, I was in a bit of a rush. Fixed now.